boards/arduino-mega2560/Makefile.include

```diff
@@ -1,5 +1,3 @@
-USEMODULE += boards_common_arduino-atmega
```

Comment:
did this one got lost? or did I overlooked it?

Reply:
no idea, did you change this recently ?